Packs/SumoLogic_Cloud_SIEM/Integrations/SumoLogicCloudSIEM/SumoLogicCloudSIEM.py

```diff
@@ -663,14 +713,153 @@ def threat_intel_update_source(client: Client, args: Dict[str, Any]) -> CommandR
+    incident_id = parsed_args.data['id']
+    if not is_inmirrorable_object(insight_id):
+        demisto.info(f"Not an Insight incident: {insight_id} so won't mirror")
+        return insight_id
+
+    if parsed_args.incident_changed and delta:
+        demisto.debug(f'Got the following delta keys {str(list(delta.keys()))} to update incident \
+            corresponding to Insight {insight_id}')
+        demisto.debug(f'Got the following delta {delta} to update incident corresponding to Insight {insight_id}')
+        demisto.debug(f"Incident Id: {incident_id}")
+        changed_data = {field: '' for field in OUTGOING_MIRRORED_FIELDS}
+        for field in delta:
+            if field in OUTGOING_MIRRORED_FIELDS:
+                changed_data[field] = delta[field]
+
+        if 'closeReason' in delta:
+            if parsed_args.inc_status == IncidentStatus.ACTIVE:
+                changed_data['status'] = 'inprogress'
+            if parsed_args.inc_status == IncidentStatus.PENDING:
+                changed_data['status'] = 'new'
+            # Close Insight if relevant
+            if parsed_args.inc_status == IncidentStatus.DONE and params.get('close_insight'):
+                demisto.debug(f'Closing Sumo Logic Insight {insight_id}')
+                changed_data['status'] = 'closed'
+            # XSOAR has by default: False Positive, Duplicate, Resolved and Other which can be
+            # mapped directly to Sumo Logic default resolutions (except for Other). For any custom
+            # resolution, there must be a 1-1 mapping between Sumo SIEM and XSOAR side.
+            reason = delta['closeReason']
+            if (reason == 'Other'):
+                reason = 'Resolved'
+            changed_data['resolution'] = reason
+            changed_data['insight_id'] = insight_id
+            demisto.debug(f'Sending update status request to Sumo Logic for Insight {insight_id}, data: {changed_data}')
+
+            insight_add_comment(client, {'insight_id': insight_id, 'comment':
+                                f"Close since the corresponding XSOAR Insight incident: {incident_id} was closed"})
+            insight_obj = insight_set_status(client, changed_data).outputs  # type: ignore
+            return insight_obj.get('readableId')  # type: ignore
+    else:
+        demisto.debug(f'Incident corresponding to Sumo Logic Insight {insight_id} was not changed.')
+
+    return insight_id
+
+
+def get_modified_remote_data_command(client: Client, args: Any) -> None:
+    ''' Gets all Sumo Logic Insights that have changed since a given time. Currently not used
+    since Sumo Logic API does not allow filtering insights by update time
+
+    Args:
+        client: Client object to call a Sumo Logic SIEM
+        args (dict): The command arguments
+
+    Returns:
+        GetModifiedRemoteDataResponse: The response containing the list of ids of Insights changed
+    '''
+    raise NotImplementedError('get-modified-remote-data not implemented')
```

Comment:
Why does this only raise exceptions?

Reply:
Hi @yucohen , I followed this https://xsoar.pan.dev/docs/integrations/mirroring_integration#update-remote-system "If the command is not implemented, make sure to raise NotImplementedError so get-remote-data will be called instead" 
And the reason I don't implement get-modified-remote-data is we currently don't have an API on Sumo Logic side to retrieve modified Insights only.